src/components/Header.html

```diff
@@ -22,15 +21,15 @@ <h4><a href="./HomePage.html" target="iframe-main-container">Vision Quiz</a></h4
       </form>
       
 <div class="iconsContainer flex-evenly">
-<a class="headerAnchorTag flex-column-center icons-btn-hover buttonHoverShadow" href="#"  target="#">  <i class="material-icons material-icons-text">person </i><span class="icon-inner-txt">Account</span> </a>
+<a class="headerAnchorTag flex-column-center icons-btn-hover buttonHoverShadow">  <i class="material-icons material-icons-text">person </i><span class="icon-inner-txt">Account</span> </a>
```

Comment:
Why have you remove the href attributes?

Reply:
I don't assign value to the button on the header file it will redirect to me elsewhere.